recipe/meta.yaml

```
@@ -28,7 +28,7 @@ requirements:
     - wheel
   run:
     - python
-    - asgiref >=3.8.1
+    - asgiref >=3.8.1,<4
     - sqlparse >=0.3.1
```

```suggestion
    - sqlparse >=0.3.1
    # the conda python package depends on tzdata
    #- tzdata  # [win]
```
https://github.com/django/django/blob/69bf08e3a32492998871eb91ad84b3c8d8117180/pyproject.toml#L12

---

I don't think this is necessary because it's a comment and seem to contradict the patch listed above.